src/shared/api.ts

```diff
@@ -334,8 +334,8 @@ export class API {
     return this.fetch('rest/startScan')
   }
 
-  async scrobble(id: string): Promise<void> {
-    return this.fetch('rest/scrobble', { id })
+  async scrobble(id: string, submission = true): Promise<void> {
```

Comment:
`submission` is `true` by default

https://web.archive.org/web/20220707025402/https://www.subsonic.org/pages/api.jsp#scrobble